include/deal.II/multigrid/mg_transfer_matrix_free.h

```diff
@@ -369,6 +389,98 @@ class MGTransferBlockMatrixFree : public MGTransferBase<LinearAlgebra::distribut
 //------------------------ templated functions -------------------------
 #ifndef DOXYGEN
 
+
+template <int dim, typename Number>
+template <typename Number2, int spacedim>
+void
+MGTransferMatrixFree<dim,Number>::
+interpolate_to_mg(const DoFHandler<dim,spacedim> &dof_handler,
+                  MGLevelObject<LinearAlgebra::distributed::Vector<Number>> &dst,
+                  const LinearAlgebra::distributed::Vector<Number2> &solution) const
```

Comment:
Apart from the issues raised in my previous general comment, the output `dst` should be the last argument of the function.

Reply:
to be fair, that's how all `copy_to_mg`, `copy_to_mg_add` and `copy_from_mg_add` are written. So I prefer to keep it consistently within this class, although that not how we usually do it in dealii

Reply:
You're right, I forgot about that. I agree that we should keep it this way, then.